code/modules/shuttle/emergency.dm

```diff
@@ -585,11 +588,40 @@
 				var/destination_dock = "emergency_away"
 				if(is_hijacked() || elimination_hijack())
 					// just double check
-					SSmapping.lazy_load_template(LAZY_TEMPLATE_KEY_NUKIEBASE)
-					destination_dock = "emergency_syndicate"
-					minor_announce("Corruption detected in \
-						shuttle navigation protocols. Please contact your \
-						supervisor.", "SYSTEM ERROR:", sound_override = 'sound/misc/announce_syndi.ogg')
+					if(hijacker.mind)
+						var/datum/mind/hijacker_mind = hijacker.mind
+
+						var/destination = hijacker_mind.get_desired_hijack_destination()
+
+						// To add another destination, go to the _DEFINES\antagonists to name your desitnation and give it a number to define it.
+						// Remember to add it to your antagonist's hijack_location varible as well.
+						// Then define where you want the shuttle to dock by by choosing or creating a new docking port in switch statement
+						// get_desired_hijack_destination() takes only the destination with the highest number in the case of an mind with multiple antagonist datums
+						switch(destination)
+							if(DEEP_SPACE)
+								var/x = 32 // In the middle so we don't have to worry about clipping the edge of the z-level.
```

Comment:
This could maybe also be delegated to a theroetical hijack datum, I don't think expanding this as an ever-growing switch statement with large blocks of code inside it would be very nice actually.